modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,297 @@
+
+| `toc_depth`                         | `tableofcontents_depth`          | Option                              | NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                           | see xref:new-and-improved-plugins-table-of-contents[Table of Contents]
+
+| `toc_header`                        | `tableofcontents_header`         | Option                              | NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].                                                                                                                                                                                                           | see xref:new-and-improved-plugins-table-of-contents[Table of Contents]
+|===
+
+- *Commands* are what is passed via the `theeditor.execCommand()` API.
++
+Where a Command name has change, calls to `theeditor.execCommand()` API must be changed to match the new name.
++
+- Configuration *Options* are what is passed when initialising the {productname} editor via `tinymce.init`.
++
+Where an Option name has changed, configurations using that option must be changed to match the new name.
++
+- *Menu items* and *Toolbar items* are *Options* from `tinymce.init` for UI and UX features, such as the {productname} Menu, Toolbar and Contextual Menu.
+
+// end::things-we-renamed[]
+
+// tag::default-value-changes[]
+[[default-value-changes]]
+== Default value changes
+
+[cols="1,1,1,1,1"]
+|===
+| Element                       | Old value                   | New value | Notes                                                                  | For more information
+
+| assignment operator character | `:`                         | `~`       | Changed in the `valid_elements` and `extended_valid_elements` schemata |
+
+| `config.height`               | `200px`                     | `400px`   | Changed to improve user experience.                                    | see xref:ui-and-ux-elements-and-components-editor-text-entry-height[Height]
+
+| `element_format`              | _no default value assigned_ | `html`    | Changed as part of modernising {productname}’s default behavior.      |
+
+| `link_default_protocol`       | `http`                      | `https`   | Changed as part of modernising {productname}’s default behavior.      |
```

Comment:
```suggestion
| `element_format`              | _no default value assigned_ | `'html'`    | Changed as part of modernising {productname}’s default behavior.      |

| `link_default_protocol`       | `'http'`                      | `'https'`   | Changed as part of modernising {productname}’s default behavior.      |
```
These options requires a string, so we probably need to make it look like a string.